pkg/executor/infoschema_reader.go

```diff
@@ -2095,12 +2095,17 @@ func (e *memtableRetriever) setDataForTiKVRegionStatus(ctx context.Context, sctx
 	}
 
 	tableInfos := tikvHelper.GetRegionsTableInfo(allRegionsInfo, is, nil)
+	virtualSchemas := []string{"information_schema", "metrics_schema", "performance_schema"}
```

Comment:
I found we already defined some variables for these database names

https://github.com/pingcap/tidb/blob/b9c54c4019636174f6c1d984344809f4499ad860/pkg/util/misc.go#L183-L189

So you can use `virtualSchemas := []string{util.InformationSchemaName.L, ... }`